httpwrapper/httpwrapper.go

```diff
@@ -0,0 +1,107 @@
+/*
+Package httpwrapper contains a small wrapper for go's native http methods.
+
+Copyright © 2020 Dusan Perkovic <dusan.perkovic@axoniq.io>
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package httpwrapper
+
+import (
+	"bytes"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"time"
+
+	"github.com/spf13/viper"
+)
+
+const tokenKey = "AxonIQ-Access-Token"
+
+var token string
+
+func init() {
+	token = viper.GetString("token")
+}
+
+// GET - Executes a GET on the given URL.
+func GET(url string) []byte {
+	req, err := http.NewRequest("GET", url, nil)
+	if err != nil {
+		log.Fatal("Error reading request ", err)
+	}
+
+	req.Header.Set(tokenKey, token)
+	client := &http.Client{Timeout: time.Second * 10}
+	resp, err := client.Do(req)
+	if err != nil {
+		log.Fatal("Error reading response. ", err)
+	}
+	defer resp.Body.Close()
+
+	responseBody, err := ioutil.ReadAll(resp.Body)
+	if err != nil {
+		log.Fatal("Error reading response body. ", err)
+	}
+
+	return responseBody
+}
+
+// POST - Executes POST on the given URL, with the given body
+func POST(url string, requestBody []byte) []byte {
+	req, err := http.NewRequest("POST", url, bytes.NewBuffer(requestBody))
+	if err != nil {
+		log.Fatal("Error reading request.", err)
+	}
+
+	req.Header.Set(tokenKey, token)
```

Comment:
I know we were using the token everywhere but not always it is needed... might create an issue to have methods with and without token.

Reply:
https://github.com/lfgcampos/axon-server-cli/issues/25